style.css

```diff
@@ -1 +1,280 @@
+    transition-duration: 0.25s;
+}
+.menu__item {
+    background-color: lightslategray;
+}
+
+
+
+/* desktop navigation */
+.desktop {
+    display: none;
+}
+
+/* hero section */
+.hero {
+    width: 90%;
+    margin: auto;
+    padding: 7rem 0 2rem 0;
+}
+.article {
+    margin-bottom: 1.5rem;
+}
+.welcome {
+    font-size: 2rem;
+    text-align: left;
+    margin-bottom: 0.5rem;
+   
+
+}
+.text_content {
+    text-align: justify;
+    font-size: 1.1rem;
+}
+.hero_img {
+    width: 100%;
+    height: 220px;
+    background-color:lightgrey;
+margin-left: 1rem;
+
+}
+.hero_img:hover {
+
+transform: scale(1.3);
+        z-index: 2;
+}
+
+
+
+/* thumbnails */
+.thumbnails {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    width: 90%;
+    margin: auto;
+    gap: 1rem;
+
+    
+}
+.thumbnail {
+    width: 12rem;
+    height: 12rem;
+    background-color:lightgray;
+    margin-left: 1rem;
+    }
+
+    .thumbnail:hover {
+        transform: scale(1.3);
+        z-index: 2;
+    }
+        
+
+/* footer */
+footer {
+    background-color: #bfc0c0;
+    margin-top: 1.5rem;
+    padding: 1rem 0;
+}
+.footer_links {
+    width: 90%;
+    margin: 1rem auto;
+}
+.footer_content {
+    width: 120px;
+    height: 16px;
+    background-color: #777777;
+    margin-top: 1rem;
+}
+.copyright {
+    width: 160px;
+    height: 16px;
+    background-color: rgb(160, 152, 152);
+    margin: 2rem auto;
+}
+
+/* tablets */
+@media screen and (min-width: 540px) {
+    .hero_img {
+        height: fit-content;
+    }
+    .thumbnail {
+        height: 18rem;
+    }
+}
+
+/* desktop */
+@media screen and (min-width: 900px) {
+    .desktop {
+        display: block;
+        padding-top: 8rem;
+    }
+    .hamburger-menu {
+        display: none;
+    }
+    .header {
+        margin-bottom: 10rem;
+    }
+    .nav_list {
+        display: flex;
+        justify-content: right;
+        width: 90%;
+        margin: auto;
+        background-color: #bfc0c0;
+        padding: 1rem 0;
+        gap: 0.5rem;
+    }
+    .nav_list li {
+        list-style-type: none;
+    
+    }
+    .nav_list li:hover{
+        transform: scale(1.3);
+        z-index: 2;
+        
+    }
+    .nav_link {
+        padding: 0.25rem 1.5rem;
+        background-color: #616161;
+        color: white;
+        text-transform: uppercase;
+        text-decoration: none;
+        font-size: small;
+    }
+    .nav_link:last-child {
+        margin-right: 1rem;
+    }
+    .hero {
+        display: flex;
+        flex-direction: row-reverse;
+        gap: 1rem;
+        padding: 0;
+        margin: 1.5rem auto;
+    
+    }
+    .article {
+        width: 50%;
+    }
+    .hero_img {
+        width: 50%;
+        height: auto;
+        
+        
+    }
+    .thumbnails {
+        grid-template-columns: repeat(4, 1fr);
+    }
+    .thumbnail {
+        height: 13rem;
+    }
+    .footer_links {
+        display: flex;
+        justify-content: center;
+        gap: 1.5rem;
+    }
```

Comment:
I would move this outside the media query, that way the footer elements are nicely centered on the mobile versions too